lib/appium_lib/android/helper.rb

```diff
@@ -152,7 +152,7 @@ def get_android_inspect class_name=false
     # @return [void]
     def page opts={}
       class_name = opts.is_a?(Hash) ? opts.fetch(:class, nil) : opts
-      puts get_android_inspect class_name
+      Appium::Logger.info get_android_inspect class_name
```

Comment:
This exists only to output to stdout, if you want to use it without messages then invoke get_android_inspect.